sim/mage/TestFrost.results

```diff
@@ -455,7 +455,7 @@ dps_results: {
 dps_results: {
  key: "TestFrost-AllItems-RobeoftheElderScribes-28602"
  value: {
-  dps: 1656.125661967007
+  dps: 1653.905027551007
  }
```

Comment:
Any hunch as to why this is changing some test results? I'm concerned there's some edge case where auras are not getting cleaned up when they should, due to the minExpires check.

Reply:
The changes are due to the iteration order change, and didn't change afterwards: for rogues and warriors I suspect mongoose procs might be affected, since MH and OH share the same RandomFloat-label, if I'm not mistaken.

Reply:
Ok that's fine then.

Reply:
FWIW, I've validated that the set of auras to expire or keep at the start of advance() were actually expired or kept afterwards. That only broke for the Adrenaline Rush expires -> extra tick -> extra special attack -> mongoose proc chain, where (correctly though) an extra aura appears.